qa/suites/rgw/notifications/frontend/civetweb.yaml

```diff
@@ -0,0 +1,3 @@
+overrides:
+  rgw:
+    frontend: civetweb
```

Comment:
you've made copies of several files that are meant to be symlinks. for example, under suites/rgw/verify/, frontend is a symlink to .qa/rgw_frontend (where .qa is symlink to ../.qa all the way up to /ceph/qa). this 'frontend' symlink lets us add or remove files in /ceph/qa/rgw_frontend and have those changes apply to every suite

the 'objectstore' and 'rgw_pool_type' symlinks works similarly - we shouldn't be copying any of those directly into this new suite

for a test suite that is specific to notifications, i don't really think it's valuable to test against multiple objectstores or pool types. that will just inflate the number of jobs in the suite without much benefit

Reply:
@cbodley as we discussed over the call. the frontend and the type of object pool, probably don't impact that test (the persistent queues are using the "log" pool).

Reply:
in the standup, we agreed to run against a single configuration instead of supporting multiple frontends/pool types/etc

to generate the proper symlinks for this configuration, i ran these commands:
```
$ cd ceph/qa/suites/rgw/notifications
$ ln -s ../.qa .
$ ln -s .qa/rgw_frontend/beast.yaml .
$ ln -s .qa/objectstore/bluestore-bitmap.yaml .
$ ln -s .qa/clusters/fixed-2.yaml .
$ ln -s .qa/distros/supported-random-distro\$ .
$ git add .qa beast.yaml bluestore-bitmap.yaml fixed-2.yaml supported-random-distro\$
$ git rm -rf clusters frontend objectstore rgw_pool_type centos_latest.yaml
```
i pushed this branch with an extra commit to https://github.com/cbodley/ceph/commits/wip-39139-symlinks. feel free to cherry-pick that into the PR

Reply:
Thank you so much, I'll cherry-pick the commit.